specification/gedcom.md

```diff
@@ -398,58 +398,35 @@ For FamilySearch GEDCOM 7.0, this is the `GEDC` structure described in [Chapter
-
-Two forms of **extension structures** are permitted:
-
-- A **tagged extension structure** is a structure whose tag matches production `extTag`. Tagged extension structures may appear as records or substructures of any other structure.
-- An **extended-use standard structure** is a structure whose type, tag, and meaning are defined in this document and whose superstructure is a tagged extension structure.
-
-Extension structures may have substructures, which may be either tagged extension structures of extended-use standard structures.
-
-All other non-standard structures are prohibited. Examples of prohibited structures include, but are not limited to,
 
-- any structure with a tag matching production `stdTag` that is not defined in this document;
-- any substructure with cardinality `{0:1}` appearing more than once;
-- a standard substructure appearing as a record or vice-versa;
-- a standard structure whose payload does not match the requirements of this document.
+##  Extensions
 
-:::note
-In some cases, an extension may need to allow multiple structures where this document allows only 1. The recommended way to do this is to create an extension tag and URI and serve a page describing how the semantics of the structure have been extended to allow multiple instances.
+The structures documented in this specification are called **standard structures**.
+Each standard structure has
 
-:::example
-Suppose I have multiple sources that give different ages of the wife at a wedding; however, this specification allows only 1 `MARR`.`WIFE`.`AGE`. An extension could not include multiple `MARR`.`WIFE` nor `MARR`.`WIFE`.`AGE`, but could define a new extension `_AGE`, give it a URL, and provide the following definition of this extension structure type at that URL:
+- A structure type, identified by a URI;
+- A set of locations where it may appear (as a record or as a substructure of a set of superstructure types), with a permitted cardinality in each location;
+- A set of substructures it may contain, with a permitted cardinality for each;
+- A payload type; if an enumeration, the set of allowed enumerated values;
+- A meaning.
 
-> Alternate age: an age attested by some source, but not accepted by the researcher as the actual age of the individual. If the age is accepted by the researcher, the standard tag `AGE` should be used instead.
+Structures must obey all of these rules to conform to this specification.
 
-This alternate age extension structure could be used as follows:
+The recommended way to go beyond the set of standard structure types in this specification or to expand their usage is to submit a feature request on the [FamilySearch GEDCOM development page](https://github.com/FamilySearch/GEDCOM/issues) so that the ramifications of the proposed addition and its interplay with other proposals may be discussed and the addition may be included in a subsequent version of this specification.
```

Comment:
This sentence, together with the text below about tagged extension structures, implies that the recommendation is to submit a specific tagged extension structure as a feature request to be included in a subsequent version, which would be read as saying that the extension structure (with literal underscore included) might appear in later versions of the official spec.  I think this should be clearer (if I have understand the intent correctly) that standard structure types will be added, but tagged extension structures will not be added to this specification.